okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -586,7 +585,7 @@ private static void retryIfAllowed(Callback callback, Call call, Exception excep
     }
 
     private static boolean isStreamingBody(Call call) {
-        return call.request().body() instanceof UnrepeatableRequestBody;
+        return call.request().body() != null && call.request().body().isOneShot();
```

Comment:
[PR](https://github.com/square/okhttp/pull/4676/commits/0f624b52bebbcac83f7cf8566dc25a694b6adbed#diff-d216814317bd8f987b94647a7bb31afaac312e9c50dd7b9bf05dcce7b1f039b9R92) removing the type and introducing the new API